sport/app/football/model/model.scala

```diff
@@ -156,6 +156,6 @@ object CompetitionDisplayHelpers {
   // It should be without effects for men competitions. For women competitions it
   // rewrite, for instance, "Norway Ladies" into "Norway Women".
```

Comment:
maybe remove this comment as it's out of date. no need to test on code after doing this

Reply:
👍